tests/Utils.test.ts

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+
+    Test for utility functions
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import * as assert from 'assert';
+import { Utils } from "../src/modules/utils/Utils";
+
+describe ('Test of isNumber, isPositiveNumber, isNegativeNumber', () =>
```

Comment:
Recommend adding separator(,) and a decimal(.) test here.

Reply:
We don't use floating-point at the momentarily.
And do not use, between numbers.
If it is used, how about adding it then?

Reply:
It's just adding a test code for a regular expression.
String can contain any character.

Reply:
I don't think it will be easy to determine the exact number of decimal places, the location of commas, etc. with a regular expression.
So I renamed the function. isInteger, isPositiveInteger, isNegativeInteger